DataFormats/TrackReco/src/TrackBase.cc

```diff
@@ -51,7 +51,8 @@ std::string const TrackBase::algoNames[] = {"undefAlgorithm",
                                             "hiRegitMuPixelLessStep",
                                             "hiRegitMuTobTecStep",
                                             "hiRegitMuMuonSeededStepInOut",
-                                            "hiRegitMuMuonSeededStepOutIn"};
+                                            "hiRegitMuMuonSeededStepOutIn",
+                                            "siStripTripletStep"};
```

Comment:
this needs to be updated as well
if the new iteration takes the enum of the `reservedForUpgrades1`, then even in this list the string needs to be updated (and this very last line added here removed, of course)

Reply:
Of course, already done